# list an arbitrary set of topics to  subscribe to and stream:
# overwrite default stream time on specific topics in this config

---
stream_time: 10  # seconds

# yamllint disable-line rule:line-length
# adds additional topics to stream, or overrides default stream time on specific topics
# leave blank to bag MIL bag script environmental variables only
topics: [["/odom", 300],
         ["/absodom", 300],
         ["/stereo/left/camera_info"],
         ["/stereo/left/image_raw", 20],
         ["/stereo/right/camera_info", 20],
         ["/stereo/right/image_raw", 20],
         ["/velodyne_points", 300],
         ["/tf", 100]]

# yamllint disable-line rule:line-length
# determines how frequently unavailable topics are resubscribed to defaults to 3.0 seconds:
resubscribe_period: 3.0  # seconds

# comment out if default env variable 'BAG_DIR'/current_date is desired:
# bag_package_path: ''

# Bool parameter that determines if current date is appended to bag directory
# for example defaults to: "home/user/bags/current-date" when True
# else bag directory goes to: "home/user/bags when False
dated_folder: true
